### PR TITLE
Bump version of syft

### PR DESCRIPTION
**Pull request by pdeslaur**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,13 +1,13 @@
 module github.com/wolfi-dev/wolfictl
 
-go 1.21
+go 1.21.0
```
Comment (luhring) [inline go.mod:3]: Why this change? Just curious

Comment (pdeslaur) [reply go.mod:3]: Not sure! I just did `go mod tidy` and it added the patch version 🤔

Comment (luhring) [reply go.mod:3]: Okay! This is the closest I found for now: https://tip.golang.org/doc/go1.21#tools 🤷
